ts/features/bonusVacanze/screens/AvailableBonusScreen.tsx

```diff
@@ -13,13 +13,14 @@ import { navigateBack } from "../../../store/actions/navigation";
 import { Dispatch } from "../../../store/actions/types";
 import { GlobalState } from "../../../store/reducers/types";
 import variables from "../../../theme/variables";
+import { maybeInnerProperty } from "../../../utils/options";
 import ActiveBonus from "../components/ActiveBonus";
 import AvailableBonusItem from "../components/AvailableBonusItem";
 import { mockedBonus } from "../mock/mockData";
 import { availableBonusesLoad } from "../store/actions/bonusVacanze";
-import { availableBonuses } from "../store/reducers/bonusVacanze";
+import { availableBonusesSelector } from "../store/reducers/availableBonuses";
 import { BonusItem, ID_TYPE_BONUS_VACANZE } from "../types/bonusList";
-import { isBonusActive } from "../utils/bonus";
+import { ID_BONUS_VACANZE_TYPE, isBonusActive } from "../utils/bonus";
```

Comment:
It seems like `ID_BONUS_VACANZE_TYPE` and `ID_TYPE_BONUS_VACANZE` are working the same way, what's the difference between them?
May we use just one of them?

Reply:
No differences 👯 
fixed in https://github.com/pagopa/io-app/pull/1857/commits/cde31d0bf3b308408ebbf4f94ed08ce6c3b76691